image_width: 720
image_height: 480
camera_name: parrot_rgbd
camera_matrix:
  rows: 3
  cols: 3
  data: [207.8449214843, 0.0, 360.0,
         0.0, 207.8449214843, 240.0,
         0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.0, 0.0, 0.0, 0.0, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0,
         0.0, 1.0, 0.0,
         0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [207.8449214843, 0.0, 360.0, 0.0,
         0.0, 207.8449214843, 240.0, 0.0,
         0.0, 0.0, 1.0, 0.0]
